### PR TITLE
feat: reply state for input bar (WEBAPP-5626)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/view_model/content/InputBarViewModel.js [hunk 1]
```diff
@@ -68,7 +68,14 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
     this.conversationHasFocus = ko.observable(true).extend({notify: 'always'});
 
     this.editMessageEntity = ko.observable();
+    this.replyMessageEntity = ko.observable();
+
+    this.replyAsset = ko.pureComputed(
+      () => this.replyMessageEntity() && this.replyMessageEntity().assets() && this.replyMessageEntity().assets()[0]
+    );
```
Comment (gregor) [inline app/script/view_model/content/InputBarViewModel.js]: Do me the favor one last time :)

File: app/script/view_model/content/InputBarViewModel.js [hunk 2]
```diff
@@ -635,6 +668,10 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       });
   }
 
+  sendMessageReply(messageText, messageEntity) {
+    //TODO: send message reply
```
Comment (atomrc) [inline app/script/view_model/content/InputBarViewModel.js]: You lazy Sloth :kissing_heart:

File: app/script/view_model/content/InputBarViewModel.js (hunk 1 above)
Comment (AndyLnd) [reply app/script/view_model/content/InputBarViewModel.js]: :) of course

**Comment by gregor** [comment]
This is pretty much like my initial approach to get the first reply to send. @atomrc  and I discussed that it would be great to add a proper state where this viemodel can either be in one of the following modes: `composing`, `editing`, `replying`.

**Pull request merged**